resources/scripts/terrain_project_importer.as

```diff
@@ -143,27 +143,40 @@ void drawUI()
         case STAGE_WRITERACES:
         {
             ImGui::Separator();
-            ImGui::TextDisabled("Writing .race files:");
+            ImGui::TextDisabled("Writing .race file:");
+			ImGui::SameLine();
             ImGui::Text(fileBeingWritten);
+			
+			// safe progressbar - dont divide by zero
+			int numRaces = int(races.raceList.length());
+			float progress = (numRaces > 0) ? float(topWrittenRace+1/numRaces) : 0.f;
```

Comment:
How about:
float(topWrittenRace+1)/float(numRaces)
or float(topWrittenRace+1)/numRaces
because if both those args are int then you could get int division and add just 0 for 1/numRaces

Reply:
I'm not following :D
I think both your suggestions would still divide by zero.

Reply:
Either way that's wery minor and I'm not going to change it.

Reply:
Ok, if you checked it and it works fine, then no need.
I just think
float(topWrittenRace+1/numRaces) does
int computation of topWrittenRace+1/numRaces with all ints
and then converts to float? If so then 1/anything would be close to zero (int division result).
Instead of float(topWrittenRace+1) first being float and surely float divided next